deployment/cdk/opensearch-service-migration/lib/service-stacks/reindex-from-snapshot-stack.ts

```diff
@@ -174,8 +180,8 @@ export class ReindexFromSnapshotStack extends MigrationServiceCore {
             mountPoints: mountPoints,
             taskRolePolicies: servicePolicies,
             cpuArchitecture: props.fargateCpuArch,
-            taskCpuUnits: 2048,
-            taskMemoryLimitMiB: 4096,
+            taskCpuUnits: props.reindexFromSnapshotWorkerSize === "maximum" ? 16 * 1024 : 2 * 1024,
```

Comment:
Recalling the conversation last week around imbalanced nodes, most use cases are better handled with horizontal scaling, so that more shards can be in progress, which give a better work distribution across target nodes.

Reply:
This is described in the options.md

Reply:
Horizontal scaling is easy to do by using the `console backfill scale [N]` command, whereas vertical scaling requires a full deployment.  I'd rather minimize CDK context changes.  If we have `maximum` as the only option, what is the risk?

Reply:
Offline recap; looks like there could be risk alongside a gap in testing coverage in this scenarios.  Keeping this as an opt-in option reduces that risk - thanks for the discussion.